projects/tuist/fixtures/ios_app_with_static_frameworks/Tests/AppTests.swift

```diff
@@ -1,10 +1,12 @@
 import Foundation
 import XCTest
+import PrebuiltStaticFramework
```

Comment:
That seems like a similar issue to the one we're trying to solve.

The PR fixes the scenario where `AppTests` transitively depends on `PrebuiltStaticFramework` but it seems we also have a bug when `AppTests` directly depends on `PrebuiltStaticFramework`. 

We should be able to define all our dependencies explicitly and have the generation logic include/exclude the appropriate frameworks rather than have us try to workaround around it 🙂

Reply:
Thanks to your changes this can be marked as resolved, correct? 😄